lib/ansible/modules/system/xfconf.py

```diff
@@ -173,6 +168,10 @@ def main():
         elif value_type is None or value_type == "":
             module.fail_json(msg='State %s requires "value_type" to be set'
                              % str(state))
+        if value.lower() == "true":
+            value = "true"
+        elif value.lower() == "false":
+            value = "false"
```

Comment:
not a blocker, but a simpler check would be

~~~python
if value.lower() in ('true', 'false'):
    value = value.lower()
~~~